certbot/src/certbot/_internal/tests/plugins/webroot_test.py

```diff
@@ -314,7 +314,7 @@ def setUp(self):
         self.path = tempfile.mkdtemp()
         self.parser = argparse.ArgumentParser()
         self.parser.add_argument("-d", "--domains",
-                                 action="append", default=[])
+                                 action=cli._DomainsAction, default=[])
```

Comment:
i love that making this a more real test is so simple and reliably catches the bug. great idea imo :clap:

part of me wants to rename `_DomainsAction` to `DomainsAction` as using code "private" to the `cli` module is a bit hacky here, but i'm honestly not too worried about that right now. i'm just happy to have a real fix with tests here